### PR TITLE
Abductor and HFZ Changes

### PR DESCRIPTION
**Pull request by SarmentiCampbell**
## About The Pull Request
Removed the immunity of Injuries and delimbs from Abductors and made it so Zombies[not the infectious one, the high functioning one] are no longer space-proof

Unfortunately, Since abductors are not only a round-start specie but  also a antag, they were also affected by the nerf and i cannot do a subtype due to the fact that round-start species are server-sided.

## Why It's Good For The Game
Maybe turning a antag specie to a round-start specie was a bad idea since y'know, It had no respiration, the inability to be dismembered or gutted, no bloods and immunity to injuries, but y'know. idk.
Also, I was told that HFZ could have a chance to be added in if their space-proof were removed since their other strength and weaknesses were fine, so here you go.

## Changelog

:cl:
balance: removed abductor's wounds immunity and delimbs.
balance: Normal Zombies, not infectious types are no longer immune to pressure.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species_types/abductors.dm [hunk 1]
```diff
@@ -31,8 +33,9 @@
 
 
 /datum/species/abductor/get_physical_attributes()
+	//bubber edit - Abductor Nerf
 	return "Abductors do not need to breathe, eat, do not have blood, a heart, stomach, or lungs and cannot be infected by human viruses. \
-		Their hardy physique prevents their skin from being wounded or dismembered, but their chunky tridactyl hands make it hard to operate human equipment."
+	Their chunky tridactyl hands make it hard to operate human equipment."
```
Comment (Majkl-J) [inline code/modules/mob/living/carbon/human/species_types/abductors.dm:38]: Override this modularly by redefining /datum/species/abductor/get_physical_attributes() in modular files

File: code/modules/mob/living/carbon/human/species_types/zombies.dm [hunk 2]
```diff
@@ -19,8 +19,10 @@
 		TRAIT_LIVERLESS_METABOLISM,
 		TRAIT_RADIMMUNE,
 		TRAIT_RESISTCOLD,
+		/* Bubber Edit - Nerf Zombie being spaceproof
 		TRAIT_RESISTHIGHPRESSURE,
 		TRAIT_RESISTLOWPRESSURE,
+		*/
```
Comment (Majkl-J) [inline code/modules/mob/living/carbon/human/species_types/zombies.dm:25]:
```
/datum/species/zombie/New()
	inherent_traits -= list(
		TRAIT_RESISTHIGHPRESSURE,
		TRAIT_RESISTLOWPRESSURE,
	)
	. = ..()
```

File: code/modules/mob/living/carbon/human/species_types/abductors.dm [hunk 3]
```diff
@@ -9,8 +9,10 @@
 		TRAIT_VIRUSIMMUNE,
 		TRAIT_NOBLOOD,
 		TRAIT_CHUNKYFINGERS_IGNORE_BATON,
+		/* Bubber Edit - Abductor Nerf
 		TRAIT_NODISMEMBER,
 		TRAIT_NEVER_WOUNDED
+		*/
 	)
```
Comment (Majkl-J) [inline code/modules/mob/living/carbon/human/species_types/abductors.dm:16]: Pretty sure this messes up with the actual abductors. Could you maybe make it its own inherited species datum and make this one not available for crew.

**Review by Majkl-J — CHANGES_REQUESTED** [review]

**Review by Majkl-J — APPROVED** [review]
Looks good, will need @StrangeWeirdKitten to update the configs to reflect this change as they merge this

**Comment by nevimer** [comment]
> Looks good, will need @StrangeWeirdKitten to update the configs to reflect this change as they merge this

I handle config. This looks fine for now.

**Review by nevimer — APPROVED** [review]

**Comment by destrucktoid** [comment]
I'd honestly rather you just remove abductors from roundstart species, they are rarely used and as cool as they are and as much as i enjoy playing Ayyde, i'd rather not see the antag get cucked because someone on skyrat who doesn't care about balance added them to roundstart. Either that or make a new version of the abductor species but this one with the nerfs suggested in this PR (assuming that's possible).

**Comment by destrucktoid** [comment]
> > I'd honestly rather you just remove abductors from roundstart species, they are rarely used and as cool as they are and as much as i enjoy playing Ayyde, i'd rather not see the antag get cucked because someone on skyrat who doesn't care about balance added them to roundstart. Either that or make a new version of the abductor species but this one with the nerfs suggested in this PR (assuming that's possible).
> 
> it is already how it's gonna work.

So this already just affects roundstart abductors and not antag abductors?

**Comment by Majkl-J** [comment]
> So this already just affects roundstart abductors and not antag abductors?

Yup, I even specifically requested it to be as such for this exact reason.

**Comment by destrucktoid** [comment]
> > So this already just affects roundstart abductors and not antag abductors?
> 
> Yup, I even specifically requested it to be as such for this exact reason.

Ah my bad then, see you've got this well in hand

**Pull request merged**